backend/src/controllers/tasks/index.ts

```diff
@@ -74,13 +75,43 @@ export const updateTask = async (req: Request, res: Response) => {
   }
 };
 
-export const getProject = async (req: Request, res: Response) => {
+export const assignTaskToUser = async (req: Request, res: Response) => {
   const { project_id } = req.params;
+  const { assignee, title, description, status, completed, priority, due_date } = req.body; // Assuming the task details are in the request body
+
   try {
-    const project = await db.select('*').from('projects').where({ id: project_id }).first();
-    project
-      ? res.status(StatusCodes.OK).json(project)
-      : res.status(StatusCodes.NOT_FOUND).json({ error: 'Project not found' });
+    const projectUsers = await db('project_user_relation')
+      .select('user_uid')
+      .where('project_id', project_id);
+
+    const isUserMember = projectUsers.some((user) => user.user_uid === assignee);
+
+    if (isUserMember) {
+      const assigneeData = await db('users')
+        .select('first_name', 'last_name')
+        .where('uid', assignee)
+        .first();
+
+      if (assigneeData) {
+        await db('tasks').insert({
+          assignee: `${assigneeData.first_name} ${assigneeData.last_name}`,
```

Comment:
Probably a better idea to use the users uid, as it is constant. Let's say a user want to update their name in the future because of a spelling mistake, then you wouldnt have to update all the lists.